components/slider/slider.js

```diff
@@ -0,0 +1,9 @@
+const slider = document.querySelector(".slider");
```

Comment:
I'm not sure but wouldn't it be important to use this piece of event listener to all the slider classes??

Reply:
Ohh that's right I didn't realize that. I will make the changes. Thanks, shubendu!!